src/PresentationalComponents/CreateImageWizard/WizardStepImageOutput.scss

```diff
@@ -0,0 +1,26 @@
+.tiles {
+    display: flex;
+}
+
+.tile {
+    flex: 1 0 0px;
+}
+
+.pf-c-tile:focus {
+    --pf-c-tile__title--Color: var(--pf-c-tile__title--Color);
+    --pf-c-tile__icon--Color: var(---pf-global--Color--100);
+    --pf-c-tile--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+    --pf-c-tile--before--BorderColor: var(--pf-global--BorderColor--100);
+}
+
+.pf-c-tile.pf-m-selected:focus {
+    --pf-c-tile__title--Color: var(--pf-c-tile--focus__title--Color);
+    --pf-c-tile__icon--Color: var(--pf-c-tile--focus__icon--Color);
+    --pf-c-tile--before--BorderWidth: var(--pf-c-tile--focus--before--BorderWidth);
+    --pf-c-tile--before--BorderColor: var(--pf-c-tile--focus--before--BorderColor);
+}
+
+.provider-icon {
+    width: 1em;
+    height: 1em;
+}
```

Comment:
Nitpick but, newline at the end of the file pliz :)

Reply:
fixed